eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/AsseticPass.php

```diff
@@ -25,6 +25,7 @@ public function process(ContainerBuilder $container)
 
         $assetFactoryDef = $container->findDefinition('assetic.asset_factory');
         $assetFactoryDef
+            ->setClass('eZ\Bundle\EzPublishCoreBundle\Assetic\AssetFactory')
```

Comment:
So we were replacing the class for some services by overriding the class parameter, and we are switching to `setClass`. Alright.